python/lbann/util/class_generator.py

```diff
@@ -96,7 +96,11 @@ def export_proto(self):
             message = getattr(proto, base_field_name)
             message.SetInParent()
         else:
-            proto = getattr(lbann_pb2, message_name)()
+            proto_modules = set([callbacks_pb2, layers_pb2, metrics_pb2, model_pb2, objective_functions_pb2, optimizers_pb2, weights_pb2])
```

Comment:
Maybe include a comment indicating that this needs to be updated if we add additional protobuf definition files.

Reply:
Done.